ci/prow/boskos/set_permissions.sh

```diff
@@ -16,6 +16,7 @@
 
 set -e
 
+readonly CUSTOM_ROLE_NAME="KnativeTests"
```

Comment:
1. This name is too generic, please use a more descriptive name.
1. This is copying the custom role from 1 project to the others. This is a single point of failure. Instead, craft a custom role YAML (write a script to generate such YAML if necessary).

With that in place, your role update can be a single command line to copy & paste instead of another script.